release.sh

```diff
@@ -0,0 +1,235 @@
+#!/bin/bash
```

Comment:
I think it would help readers to have a short comment at the beginning of this file explaining its purpose. Since there are a lot of Apache-specific things here, it might also be nice to link to the relevant Apache regulations and documentations. This will help other maintain the script and fix issues that might arise.